doc/developers/contributing.rst

```diff
@@ -228,7 +228,7 @@ then submit a "pull request" (PR):
    changes::
 
        $ git fetch upstream
```

Comment:
This line is now redundant. Please make sure the new branch starts at the upstream master.